core/src/main/java/com/xmartlabs/bigbang/core/controller/EntityController.java

```diff
@@ -52,7 +56,7 @@
                 .toFlowable()
         )
         .subscribeOn(Schedulers.io())
-        .observeOn(Schedulers.io())
+        .observeOn(Schedulers.io(), true)
```

Comment:
delayError: indicates if the onError notification may not cut ahead of onNext notification on the other side of the scheduling boundary. If true a sequence ending in onError will be replayed in the same order as was received from upstream